test/e2e-test/src/index.ts

```diff
@@ -515,16 +515,16 @@ class Demo {
 
     console.log("keyReleasePolicy response: ", keyResponse);
     Demo.assert(
-      `keyResponse["x-ms-sevsnpvm-smt-allowed"][0] === true`,
-      keyResponse["x-ms-sevsnpvm-smt-allowed"][0] === true,
+      `keyResponse.claims["x-ms-sevsnpvm-smt-allowed"][0] === true`,
+      keyResponse.claims["x-ms-sevsnpvm-smt-allowed"][0] === true,
     );
     Demo.assert(
-      `keyResponse["x-ms-ver"][0] === '2'`,
-      keyResponse["x-ms-ver"][0] === "2",
+      `keyResponse.claims["x-ms-ver"][0] === '2'`,
+      keyResponse.claims["x-ms-ver"][0] === "2",
     );
     Demo.assert(
-      `keyResponse["x-ms-sevsnpvm-is-debuggable"][0] === false`,
-      keyResponse["x-ms-sevsnpvm-is-debuggable"][0] === false,
+      `keyResponse.claims["x-ms-sevsnpvm-is-debuggable"][0] === false`,
+      keyResponse.claims["x-ms-sevsnpvm-is-debuggable"][0] === false,
```

Comment:
How about having some basic test cases for gt and gte?

Reply:
we have many in the unit tests

Reply:
done